examples/si5351_serial_console/si5351_serial_console.ino

```diff
@@ -0,0 +1,717 @@
+/*
+ * si5351mcu - Si5351 library for Arduino, MCU tuned for size and click-less
+ *
+ * Serial Console Test Harness.
+ *
+ * Copyright (C) 2017 Pavel Milanes <pavelmc@gmail.com>
+ *
+ * Many chunk of codes are derived-from/copied from other libs
+ * all GNU GPL licenced:
+ *  - Linux Kernel (www.kernel.org)
+ *  - Hans Summers libs and demo code (qrp-labs.com)
+ *  - Etherkit (NT7S) Si5351 libs on github
+ *  - DK7IH example.
+ *  - Jerry Gaffke integer routines for the bitx20 group
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+ 
+/* 
+ * Serial Console Test Harness code added
+ *    2021-10-01 - Iain W. Bird, 2E0WZR
+ *    
+ */
+
+/* 
+ * CAUTION:  As this is for testing purposes only, there is little 
+ *           validation or error checking.
+ * 
+ */
+ 
+/***************************************************************************
+ * Type HELP into the Arduino Serial Console line for a list of commands.
+ *
+ * Take into account your XTAL error, see Si.correction(###) below
+ ***************************************************************************/
+
+#include "si5351mcu.h"
+
+// lib instantiation as "Si"
+Si5351mcu Si;
+
+int   cmdHelp( void );
+void  resetInput( void );
+float getPllFraction( uint8_t pllIndex );
+void  printDecimalByte( int val );
+
+uint8_t devicePresent = 0;
+
```

Comment:
Change uint8_t devicePresent = 0; to uint8_t devicePresent = 1; and it will all be fine.  This for #6, which is not done.